app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -101,10 +101,9 @@ class ChatActivity : AppCompatActivity(), IChatView {
         setUpUI()
         initializationMethod(firstRun)
 
-        example = if (intent.getStringExtra("example") != null) {
-            intent.getStringExtra("example")
-        } else {
-            ""
+        if (intent.hasExtra("example")) {
+            example = intent.getStringExtra("example")
+            intent.removeExtra("example")
```

Comment:
Why?

Reply:
Sir, during debugging I found that the app was crashing because of an infinite loop caused by the presence of the intent extra value example. When we press the try button, simply an intent to open chat activity is called with an intent parameter "example" that contains the query value. Here, the value of the intent was not being reset, after taking the value from it. SO, the app was having an infinite loop, which caused it to crash.